templates/admin/config.tmpl

```diff
@@ -184,7 +184,7 @@
 				{{if .MailerEnabled}}
 					<dt>{{.i18n.Tr "admin.config.mailer_name"}}</dt>
 					<dd>{{.Mailer.Name}}</dd>
-					{{if not .Mailer.UseSendmail}}
+					{{if ne .Mailer.MailerType "sendmail"}}
```

Comment:
Should it better be eq mail/smtp?

Reply:
probably should be multiple if/elseif to build out mailer types, however I was aiming for parity in this PR rather than enhancement.

I can fix it in the morning, but I'm off to sleep now. Feel free to push to my branch if you would like.

Reply:
updated